pkgs/development/python-modules/dependency-injector/default.nix

```diff
@@ -72,6 +73,10 @@ buildPythonPackage rec {
     "tests/unit/wiring/test_*_py36.py"
   ];
 
+  passthru.updateScript = nix-update-script {
+    attrPath = "python3Packages.dependency-injector";
+  };
+
```

Comment:
Python modules are usually updated with `nix-update`.

Reply:
You are right, somehow I thought to have tested that it didn't work out-of-the-box. But that was an error, removed it. Thanks :)